eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -6,37 +6,44 @@ parameters:
   MaxParallel: 0
   Matrix:
     Linux - Java 8 (AzureCloud):
-      OSVmImage: 'ubuntu-18.04'
+      Pool: azsdk-pool-mms-ubuntu-1804-general
+      OSVmImage: MMSUbuntu18.04
```

Comment:
Are these names just for documentation purposes for verifying the OS? Or do the pools need them? If they are just our own identifier is there any reason we cannot keep them like the default names so this would just be "ubuntu-18.04"?

Reply:
If provided, they need to be an actual correct image name (which is MMSUbunut18.04 in this case).